src/main/java/com/exadel/discountwebapp/vendor/service/VendorService.java

```diff
@@ -55,16 +55,16 @@ public VendorResponseVO create(VendorRequestVO request) {
 
     @Transactional(propagation = Propagation.REQUIRED)
     public VendorResponseVO update(Long id, VendorRequestVO request) {
-        vendorValidator.validate(request);
```

Comment:
Why this line with validation was deleted?

Reply:
cuz this function check if the vendor with email does already exist in DB (if exists - throws exception), so if we going to update vendor - vendor with this email  will already be in DB

Reply:
but what if we decide to update vendor's email and at the time the vendor with such email already exists in the db?

Reply:
I have added checking about existing email (we can update vendor only with already existing email in DB). But maybe we will need to change the email?( and in this case we can not do it)

Reply:
fixed it